components/solicitation/FollowSolicitation.tsx

```diff
@@ -0,0 +1,120 @@
+import { StatusType } from "$store/packs/utils/constants.ts";
+import type { AppContext } from "$store/apps/site.ts";
+import Step from "./Step.tsx";
+import Icon from "$store/components/ui/Icon.tsx";
+
+export interface ToolTip {
+  /** @title Em cima */
+  up: string;
+  /**
+   * @format textarea
+   * @title Em Baixo
+   */
+  down: string;
+}
+/** @titleBy title */
+export interface Status {
+  /** @title Título */
+  title: string;
+  /**
+   * @title Mensagem
+   * @format html
+   */
+  message?: string;
+  /** @description Texto ao passar o mouse no titulo ou no icone de etapa */
+  toolTip: ToolTip;
+  /** @title Tipo de Status */
+  statusType: StatusType;
+}
+
+export interface Props {
+  status: Status[];
+  /** @title Mensagem */
+  message?: string;
+}
+
+export const loader = async (props: Props, _req: Request, ctx: AppContext) => {
+  const statusMessage = await ctx.invoke(
+    "deco-sites/niivu-bank/loaders/actions/analysiStatus.ts",
+  );
+
+  if (typeof statusMessage !== "string") {
+    return {
+      ...props,
+      statusMessage: "Abertura de Conta",
+      isDesk: ctx.device === "desktop",
+    };
+  }
+  return { ...props, statusMessage, isDesk: ctx.device === "desktop" };
```

Comment:
Mantenha um padrão de nomeação.
```suggestion
  return { ...props, statusMessage, isDesktop: ctx.device === "desktop" };
```